### DISCUSSION TITLE
Deployments down?

### DISCUSSION THREAD

**Discussion by scruffyai** [vercel/community#2851] [Help] [answered]
### Summary

I cannot deploy valid builds. Getting this error on deploying assets step. 

```network timeout at: https://api-sfo1.vercel.com/v2/now/deployments/dpl_4vh6YdmJSpVrR6o1j5KUfypZNVYS/builds/bld_h2lewnpid/lambda?ownerId=team_mVk9aPuZOe5tk1D4tvMMkTjL&projectId=prj_Jj3lK9B9QGVJkJmB8VfwPQUrfIsy&traceCarrier=%7B%22ot-baggage-clihookAt%22%3A%221686685636261%22%2C%22x-datadog-trace-id%22%3A%224683785120523410449%22%2C%22x-datadog-parent-id%22%3A%222277778572466052638%22%2C%22x-datadog-sampling-priority%22%3A%222%22%2C%22traceparent%22%3A%2200-0000000000000000410025c0590edc11-1f9c4b8b20d5de1e-01%22%2C%22tracestate%22%3A%22dd%3Ds%3A2%22%7D&traceCarrier=%7B%22ot-baggage-clihookAt%22%3A%221686685636261%22%2C%22x-datadog-trace-id%22%3A%224683785120523410449%22%2C%22x-datadog-parent-id%22%3A%226851705173141113675%22%2C%22x-datadog-sampling-priority%22%3A%222%22%2C%22traceparent%22%3A%2200-0000000000000000410025c0590edc11-5f16258d96c84b4b-01%22%2C%22tracestate%22%3A%22dd%3Ds%3A2%22%7D&traceCarrier=%7B%22ot-baggage-clihookAt%22%3A%221686685636261%22%2C%22x-datadog-trace-id%22%3A%224683785120523410449%22%2```

### Steps to Reproduce

Try to deploy

**Comment by swarnava** [comment]
Hi there,

Thanks for reaching out to Vercel Support!

At this time we are currently experiencing issues deploying Serverless Functions to the `iad1` region, this results from an upstream provider. Please try and update your Serverless Function region to `cle1` as a workaround and redeploy: https://vercel.com/docs/concepts/functions/serverless-functions/regions. We have updated our Status Page with additional information. https://www.vercel-status.com/incidents/cqkdqrzqkspm

**Reply by scruffyai** [reply]
I am not deploying any serverless functions

**Reply by swarnava** [reply]
Hey, the lambda creation api is hitting the error which led to believe this is related to current outage. Changing region should help hopefully

**Comment by scruffyai** [comment]
Why mark this as answered? Shouldn't the person asking the question do that lol?

**Comment by swarnava** [comment, marked as answer]
Serverless invocation and deployment has back to normal. A redeployment with current configuration should work now.
